tests/phpunit/Checker/Checks/Enqueued_Scripts_In_Footer_Check_Tests.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Tests for the Enqueued_Scripts_In_Footer_Check class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\Check_Context;
+use WordPress\Plugin_Check\Checker\Check_Result;
+use WordPress\Plugin_Check\Checker\Checks\Enqueued_Scripts_In_Footer_Check;
+use WordPress\Plugin_Check\Test_Utils\TestCase\Static_Check_UnitTestCase;
+
+class Enqueued_Scripts_In_Footer_Check_Tests extends Static_Check_UnitTestCase {
+
+	public function test_run_with_errors() {
+		$enqueued_scripts_in_footer_check = new Enqueued_Scripts_In_Footer_Check();
+		$check_context                    = new Check_Context( TESTS_PLUGIN_DIR . '/tests/phpunit/testdata/plugins/test-plugin-enqueued-scripts-in-footer-check-with-errors/load.php' );
```

Comment:
After merging `trunk` back, please update this to use the new constant introduced in #189.